README.md

```diff
@@ -417,7 +417,7 @@ UACLIENT_BEHAVE_REUSE_IMAGE=your-custom-ami tox -e behave-awspro-20.04
 The following tox environments allow for testing focal on Azure:
 
 ```
-  # To test ubuntu-pro-images
```

Comment:
On PR #1947 we have decided to have the cloud names from the docs, since we infer which cloud is being used by the tox command